accumulo-iterators/src/main/java/org/vertexium/accumulo/iterator/EdgeRefFilter.java

```diff
@@ -1,21 +1,29 @@
 package org.vertexium.accumulo.iterator;
 
+import com.google.common.collect.Lists;
```

Comment:
we should not rely on guava here. Guava is transitively pulled in from Accumulo and I don't want to rely on them, I would like to keep the iterator as standalone as possible.